config/prow/cluster/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -19,9 +19,33 @@
             },
           },
           {
-            alert: 'ghproxy-global-status-code-abnormal',
+            alert: 'ghproxy-global-status-code-5xx',
             expr: |||
-              sum(rate(github_request_duration_count{status=~"[45]..",status!="404",status!="410"}[5m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) * 100 > 3
+              sum(rate(github_request_duration_count{status=~"5.."}[5m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) * 100 > 3
+            |||,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: '{{ $value | humanize }}%% of all API requests through the GitHub proxy are errorring with code {{ $labels.status }}. Check %s.' % [monitoringLink('/d/%s/github-cache?orgId=1&refresh=1m&fullscreen&panelId=8' % [dashboardID], 'the ghproxy dashboard')],
+            },
+          },
+          {
+            alert: 'ghproxy-specific-status-code-4xx',
+            expr: |||
+              sum(rate(github_request_duration_count{status=~"4..",status!="404",status!="410"}[30m])) by (status,path) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) by (path) * 100 > 10
```

Comment:
The second time interval needs to be updated to `30m` as well. Same with the other 4xx alert.

Reply:
good catch, done